packages/wdio-spec-reporter/src/index.js

```diff
@@ -171,6 +178,16 @@ class SpecReporter extends WDIOReporter {
 
                 // Output for a single test
                 output.push(`${testIndent}${this.chalk[this.getColor(state)](this.getSymbol(state))} ${testTitle}`)
+
+                // print cucumber data table cells
+                if (test.argument) {
```

Comment:
I think test argument is not or can be not necessarily cucumber table.
Can you please add some verification to ensure test.argument is actually valid cucumber table?